example/image-classification/benchmark_score.py

```diff
@@ -74,11 +75,17 @@ def score(network, dev, batch_size, num_batches):
     devs.append(mx.cpu())
 
     batch_sizes = [1, 2, 4, 8, 16, 32]
-
     for net in networks:
         logging.info('network: %s', net)
         for d in devs:
             logging.info('device: %s', d)
             for b in batch_sizes:
-                speed = score(network=net, dev=d, batch_size=b, num_batches=10)
-                logging.info('batch size %2d, image/sec: %f', b, speed)
+                for dtype in ['float32', 'float16']:
+                    if d == mx.cpu() and dtype == 'float16':
+                        #float16 is not supported on CPU
+                        continue
+                    elif net in ['inception-bn', 'alexnet'] and dt == 'float16':
```

Comment:
I guess this `dt` here doesn't work, please check again. By the way, it's necessary to add a logging here to notice user what the dtype is.

Reply:
Thanks for letting me know, fixing it here https://github.com/apache/incubator-mxnet/pull/11533/